styleguide/derek/view-templates/leadership/_leadership.scss

```diff
@@ -0,0 +1,53 @@
+.leadership-individual {
+  @include make-sm-column(3);
+  @include make-xs-column(6);
+  margin: 20px auto;
+}
+
+.leadership-image {
+  @include center-block;
+}
+
+.leadership-imageWrapper {
+  display: none;
+}
+
+.leadership-linkWrapper {
+  margin-top: 0;
+}
+
+.leadership-attrWrapper {
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.leadership-name {
+  margin-bottom: 0;
+}
+
+.leadership-title {
+  margin: 0;
+}
+
+.leadership-link {
+  @include standard-link;
+}
+
+@media only screen and (min-width: $screen-xs-max) {
+  .leadership-individual {
+    &.center-block {
+      float: none;
```

Comment:
@WidgetsBurritos My question here is are we really going to use this? How are we actually doing to be adding these to the page in drupal? I imagine they would be panels that go into layouts? I think talking through that may help solve this issue.